src/mappings/entities/mapAsset.ts

```diff
@@ -286,32 +302,73 @@ const handleAssetOwnershipTransferred = async (blockId: string, params: Codec[])
   await asset.save();
 };
 
-const handleAssetTransfer = async (blockId: string, params: Codec[]) => {
+const handleAssetTransfer = async (blockId: string, params: Codec[], event: SubstrateEvent) => {
   const [, rawTicker, rawFromPortfolio, rawToPortfolio, rawAmount] = params;
-  const { identityId: fromDid } = getPortfolioValue(rawFromPortfolio);
-  const { identityId: toDid } = getPortfolioValue(rawToPortfolio);
-  // We ignore the transfer case when Asset tokens are issued/redeemed
-  if ([fromDid, toDid].includes('0x00'.padEnd(66, '0'))) {
-    return;
-  }
-  const transferAmount = getBigIntValue(rawAmount);
+  const { identityId: fromDid, number: fromPortfolioNumber } = getPortfolioValue(rawFromPortfolio);
+  const { identityId: toDid, number: toPortfolioNumber } = getPortfolioValue(rawToPortfolio);
   const ticker = serializeTicker(rawTicker);
+  const transferAmount = getBigIntValue(rawAmount);
 
-  const asset = await getAsset(ticker);
-  asset.totalTransfers += BigInt(1);
-  asset.updatedBlockId = blockId;
+  let fromPortfolioId = `${fromDid}/${fromPortfolioNumber}`;
+  let toPortfolioId = `${toDid}/${toPortfolioNumber}`;
 
-  const [fromHolder, toHolder] = await Promise.all([
-    getAssetHolder(ticker, fromDid, blockId),
-    getAssetHolder(ticker, toDid, blockId),
-  ]);
+  const promises = [];
 
-  fromHolder.amount = fromHolder.amount - transferAmount;
-  fromHolder.updatedBlockId = blockId;
-  toHolder.amount = toHolder.amount + transferAmount;
-  toHolder.updatedBlockId = blockId;
+  if (fromDid === '0x00'.padEnd(66, '0')) {
```

Comment:
Can we assign `'0x00'.padEnd(66, '0')` to a const and use that? Something like `emptyDid`